pallets/issue/src/lib.rs

```diff
@@ -147,6 +152,25 @@ pub mod pallet {
 	pub(super) type IssueMinimumTransferAmount<T: Config> =
 		StorageValue<_, BalanceOf<T>, ValueQuery>;
 
+	#[pallet::storage]
+	pub(super) type LimitVolumeAmount<T: Config> =
+		StorageValue<_, Option<BalanceOf<T>>, ValueQuery>;
+
+	///Currency id for limit rate volume value. DOT, BTC or PEN
+	#[pallet::storage]
+	pub(super) type LimitVolumeCurrencyId<T: Config> = StorageValue<_, T::CurrencyId, ValueQuery>;
+
+	#[pallet::storage]
+	pub(super) type CurrentVolumeAmount<T: Config> = StorageValue<_, BalanceOf<T>, ValueQuery>;
+
+	/// Represent interval define regular 24 hour intervals (every 24 * 3600 / 12 blocks)
+	#[pallet::storage]
+	pub(super) type IntervalLength<T: Config> = StorageValue<_, T::BlockNumber, ValueQuery>;
+
+	// Represent current interval current_block_number / IntervalLength
+	#[pallet::storage]
+	pub(super) type CurrentInterval<T: Config> = StorageValue<_, T::BlockNumber, ValueQuery>;
```

Comment:
I thought about it and I would rename this storage item to "LastIntervalIndex". We want to store the index of the last interval where a issue/redeem occurred so that is more fitting I think.